test/runtests.jl

```diff
@@ -1,4 +1,22 @@
 using Test, Example
+using MySQL,SQLite, MySQL.API
 
 @test hello("Julia") == "Hello, Julia"
 @test domath(2.0) ≈ 7.0
+
+@testset "_dbconnect function for SQLite" begin
+
+    conn= _dbconnect(SQLite.DB, "DBConnector.jl/test/data/database.db")
+    @test @isdefined conn
+
+end
+ 
+@testset "_dbconnect function for MySQL" begin
+    
+    conn = _dbconnect(MySQL.Connection, "localhost", "debian-sys-maint","HGGsOLypO2LVqq1v", db="mydatabase", port=3306, unix_socket="/var/run/mysqld/mysqld.sock")
+    @test typeof(conn) == MySQL.Connection
+    @test isopen(conn)
+    close(conn)
+
+end
```

Comment:
This is very great that this is working! Like you said, we'll need to determine how to run these tests somewhere else. I'd like to somehow get this connection made to the GSoC Server for testing over the summer while somehow maintaining secrecy for our server credentials. Let's talk about this at our weekly check-in!